### ISSUE TITLE
placeholder ticket -- how do we make sure that individual gitcoin grants are successful?

### ISSUE THREAD

**Issue by owocki** [gitcoinco/web#3377]
this is a ticket for reserving engineering/product attention on the question "how do we make sure that individual gitcoin grants are successful?".  by enabling grants to be succeessfuly, we enable grant posters to be successful, we improve our positioning in the ecosystem

**Comment by kuhnchris** [comment]
Hmm, just thinking loud: 
- Give the supporters a special kudos for "First grant funded", "Participated in a succeesful grant" or "I GRANTed"-sticker when the grant finishes.
- Maybe a personalized mail with a personalized "thank you" message for the grant participants
- Incencitives, like posting it via gitcoinbot, maybe add a little "ad" footer on the gitcoin github messages where we select random grants to the post?
- Maybe leverage CodeFund?

**Comment by Hibero** [comment]
I'm a nobody in all of this but getting a version of kudos and having tiers with easy and highly visible buttons such as "Join this Tier for $2" would make funding a lot easier and more rewarding. I know it sounds silly but it does help. 
 [Example](https://www.patreon.com/Kurzgesagt)

Also, having pooled grants would be much easier. Such as, I want to fund all ETH 2.0 Client Implementations, so having a group that automatically splits my subscription between all of the their grants would be awesome.

**Keep up the great work! I'm really liking this new feature.**

**Comment by kuhnchris** [comment]
Oooh I love that Kurzgesagt guy, he makes awesome ELI5 videos on Youtube :-)
But yeah, what you mean would actually be "Name your tiers". I'm not entirely sure the current data model supports this, but maybe that's an idea if the demand is high enough. Basically loop over tiers, display buttons which (very basically) pre-fill the subscription amount in the subscribe form.

Regarding pooled grants - so very highlevel spoken, you'd like to use it like the "advanced payout" feature where you have a maximum amount you want to "distribute" (per month) and want to distribute it among selected grants, OR, would you rather want something like a "grant limit" where you define you'd throw in X amount per month max, and it automatically splits it up to all the grants currently subscribed to? 
The later doesn't work with the subscription model of the ERC1337 / recurring subscription model, as far as I understand that. (please correct me if i'm wrong @owocki , i'm not that strong in the contract limitations!) As I understand, that would involve canceling all the other subscriptions and creating new ones whenever you subscribe to a new grant, resulting in dozens of GWEI of gas just to throw the old subscription out, and deploy the (n+1) subscriptions.

As I said, I could be wrong, but i THINK that's how it works (YMMV).

Thanks for your ideas and keep 'em coming! :-)
Chris

**Comment by kuhnchris** [comment]
> i think theres a lot to learn from how we got @austintgriffiths first grant fully funded.
> 
> i know @ceresstation has some ideas here and is going to compile them into a 'best practices' doc for grant posters.
> 
> one of the main things i think we need to do is put the grant poster in charge of promoting their grant.. and also to give them the tools they need to ask for money.
> 
> another idea we had is rewards for different contribution levels

Do you think it makes sense to make a meeting with Austin (either within the community live stream, talking about his experience, or privately so we can discuss improvements)?

**Comment by owocki** [comment]
has everyone seen @ceresstation 's response on https://github.com/gitcoinco/web/issues/3329 ? 

> Give the supporters a special kudos for "First grant funded", "Participated in a succeesful grant" or "I GRANTed"-sticker when the grant finishes.

I like this idea.. I just asked for artwork for this

> Maybe a personalized mail with a personalized "thank you" message for the grant participants

I also like this idea.. When people sign up.. Maybe we can have a personalized message for them from the funder

TODO - write a ticket

> Maybe leverage CodeFund?

this is a good idea.  @coderberry in addition to getting attnetion for each Gitcoin Bounty, we could perhaps do it for Gitcoin Grants too!

> I'm a nobody in all of this but getting a version of kudos and having tiers with easy and highly visible buttons such as "Join this Tier for $2" or "Buy the Grantees a Drink ($6)" would make funding a lot easier and more rewarding. I know it sounds silly but it does help.

I like the idea for tiers.  we could perhaps setup a lean test of this.

TODO - write a ticket
